app/models/server_role.rb

```diff
@@ -5,32 +5,19 @@ class ServerRole < ActiveRecord::Base
   validates_presence_of     :name
   validates_uniqueness_of   :name
 
-  FIXTURE_DIR = File.join(Rails.root, "db/fixtures")
-
-  def self.seed_data
-    File.read(File.join(FIXTURE_DIR, "#{to_s.pluralize.underscore}.csv"))
-  end
-
   def self.seed
-    data = seed_data.split("\n")
-    cols = data.shift.split(",")
-
-    data.each do |a|
-      next if a =~ /^#.*$/ # skip commented lines
+    CSV.foreach(fixture_path, :headers => true, :skip_lines => /^#/).each do |csv_row|
+      action = csv_row.to_hash
+      action['license_required'] ||= ''
```

Comment:
Yes it is, but is its value displayed anywhere? Does any controller return it as a part of a response? Would the behaviour of the app change in any way if we just dropped this column from the server_roles table?